examples/text-generation/flap_pruning/readme.md

```diff
@@ -1,17 +1,17 @@
-# FLAP 0.5x Llama3.1-70b
+# Llama3.1 70B: 0.5x the cost & size
```

Comment:
this will not be 0.5x now as start layer is 56

Reply:
will infer this based on the difference in parameters.